mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactAudioView.js

```diff
@@ -0,0 +1,173 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { getSrc } from './ShowArtifactPage';
+import { getArtifactContent } from '../../../common/utils/ArtifactUtils';
+import WaveSurfer from 'wavesurfer.js';
+import './ShowArtifactAudioView.css';
+
+class ShowArtifactAudioView extends Component {
```

Comment:
Can we convert this to a functional component?

Reply:
What do you mean exactly? Turn the `<div id="waveform">` that Wavesurfer uses into its own functional component? Or do you mean to make the entire `ShowArtifactAudioView` class a functional component?

Reply:
> do you mean to make the entire ShowArtifactAudioView class a functional component?

Yes, that's what I meant.